main.tf

```diff
@@ -348,7 +348,6 @@ data "aws_iam_policy_document" "main" {
 resource "aws_s3_bucket" "aws_logs" {
   bucket        = var.s3_bucket_name
   acl           = var.s3_bucket_acl
-  region        = var.region
```

Comment:
@eeeady @mdrummerboy09 I know this is already merged, but since the region variable is not longer required we should remove it from variables.tf and document an upgrade path about it being deprecated.

Reply:
makes sense! can do and will do!